File: BeeCrowd-Python/Problem 1012.py
# Asking The Input
X, Y, Z = input().split()


# Defining Variables
A = float(X)
B = float(Y)
C = float(Z)
Pi = 3.14159

#Equations
Triangulo = (A*C)/2
Circulo = Pi*(C**2)
Trapezio = (A+B)*(C/2)
Quadrado = (B*B)
Retangulo = (A*B)

#Print
print(f"TRIANGULO: {Triangulo:.3f}")
print(f"CIRCULO: {Circulo:.3f}")
print(f"TRAPEZIO: {Trapezio:.3f}")
print(f"QUADRADO: {Quadrado:.3f}")
print(f"RETANGULO: {Retangulo:.3f}")

File: BeeCrowd - Java/Problem_1012.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner userInput = new Scanner(System.in);
        double a = userInput.nextDouble();
        double b = userInput.nextDouble();
        double c = userInput.nextDouble();
        
        
        double tri =  (a * c) / 2;
        double cir =  3.14159 * (c * c);
        double tra =  ((a + b) * c) / 2;
        double qua =  b * b;
        double ret =  a * b;
        userInput.close();
        
        System.out.printf("TRIANGULO: %.3f%n", tri);
        System.out.printf("CIRCULO: %.3f%n", cir);
        System.out.printf("TRAPEZIO: %.3f%n", tra);
        System.out.printf("QUADRADO: %.3f%n", qua);
        System.out.printf("RETANGULO: %.3f%n", ret);
        
    }
 
}